bouncer/__init__.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+import pyramid.config
+
+
+def settings():
```

Comment:
It's a small thing, but I would tend to avoid naming getters like this. If I saw a `settings` object in this module, I probably wouldn't assume it was callable. If I saw a `get_settings` object in the module, I definitely *would*.